### PR TITLE
Created r.md

### PR DESCRIPTION
**Pull request by trishitchar**
Short and concise.

### CODE REVIEW

**Comment by trishitchar** [comment]
@onecompiler-ops
Can you please review and merge.

File: r-markdown.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title:  r
```
Comment (onecompiler-ops) [inline r-markdown.md:2]: Title should be `R`
File name should be `r.md`

File: r-markdown.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+description: r  Cheat Sheet gives you a quick reference of basic r markdown language. 
+First appeared:	August 1993
+Stable release:	4.2.1 / 23 June 2022
+---
+  
+## What is ```r``` ?
+Introduction to ```r```.<br>
+```r```  is a language and environment for statistical computing and graphics. It is a GNU project which is similar to the S language and environment which was developed at Bell Laboratories (formerly AT&T, now Lucent Technologies) by John Chambers and colleagues. 
+
+```r``` provides a wide variety of statistical (linear and nonlinear modelling, classical statistical tests, time-series analysis, classification, clustering, …) and graphical techniques, and is highly extensible. The S language is often the vehicle of choice for research in statistical methodology, and ```r``` provides an Open Source route to participation in that activity.
```
Comment (onecompiler-ops) [inline r-markdown.md:12]: Can you reduce the description and increase code samples, please? This is more of a cheat sheet than like a Tutorial

**Review by onecompiler-ops — CHANGES_REQUESTED** [review]

File: r-markdown.md (hunk 2 above)
Comment (trishitchar) [reply r-markdown.md]: Issue resolved_

**Comment by onecompiler-ops** [comment]
name of the file is still wrong

**Comment by trishitchar** [comment]
Done.
Now see sir.

**Comment by onecompiler-ops** [comment]
it's still `r-markdown.md`

**Pull request closed without merging**